community-operators/cockroachdb/cockroachdb.v2.0.9.clusterserviceversion.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"charts.helm.k8s.io/v1alpha1","kind":"Cockroachdb","metadata":{"name":"example"},"spec":{"CacheSize":"25%","ClusterDomain":"cluster.local","Component":"cockroachdb","ExternalGrpcName":"grpc","ExternalGrpcPort":26257,"ExternalHttpPort":8080,"HttpName":"http","Image":"cockroachdb/cockroach","ImagePullPolicy":"Always","ImageTag":"v2.1.3","InternalGrpcName":"grpc","InternalGrpcPort":26257,"InternalHttpPort":8080,"JoinExisting":[],"Locality":"","MaxSQLMemory":"25%","MaxUnavailable":1,"Name":"cockroachdb","NetworkPolicy":{"AllowExternal":true,"Enabled":false},"NodeSelector":{},"PodManagementPolicy":"Parallel","Replicas":3,"Resources":{},"Secure":{"Enabled":false,"RequestCertsImage":"cockroachdb/cockroach-k8s-request-cert","RequestCertsImageTag":"0.4","ServiceAccount":{"Create":true,"Name":null}},"Service":{"annotations":{},"type":"ClusterIP"},"Storage":"100Gi","StorageClass":null,"Tolerations":{},"UpdateStrategy":{"type":"RollingUpdate"}}}]'
+    categories: Helm
+    description: CockroachDB
+    containerImage: quay.io/helmoperators/cockroachdb:2.0.9
+    createdAt: 2019-01-23T13-15-20Z
+    support: a-robinson
+  name: cockroachdb.v2.0.9
+  namespace: placeholder
+spec:
+  customresourcedefinitions:
+    owned:
+    - description: CockroachDB
+      displayName: cockroachdb
+      kind: Cockroachdb
+      name: cockroachdbs.charts.helm.k8s.io
+      version: v1alpha1
+  keywords:
+  - helm
+  displayName: cockroachdb
+  provider:
+    name: Helm Community
+  maturity: stable
+  version: 2.0.9
+  links:
+  - name: Helm Chart Source
+    url: https://github.com/cockroachdb/cockroach
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  maintainers:
+  - email: alex@cockroachlabs.com
+    name: a-robinson
+  selector:
+    matchLabels:
+      alm-owner-etcd: couchbaseoperator
+      operated-by: couchbaseoperator
+  labels:
+    alm-owner-etcd: couchbaseoperator
+    operated-by: couchbaseoperator
+  install:
+    spec:
+      deployments:
+      - name: cockroachdb-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: cockroachdb
+          template:
+            metadata:
+              labels:
+                name: cockroachdb
+            spec:
+              serviceAccountName: cockroachdb
```

Comment:
This should match the serviceAccount listed under `clusterPermissions` or the other way around.